### PR TITLE
Add Nextjs demo application and deploy through sst.

### PR DESCRIPTION
**Pull request by brenzy**
This PR copies the Remix demo to a Nextjs application.  Very few changes were needed to the original Remix code to get it running in Nextjs.  

The existing SST Remix deployment was changed to deploy the Next application instead.  SST uses [OpenNext](https://open-next.js.org/) to split the Next.js build, enabling deployment to  AWS Lambda.

The original Remix demo application was left for reference.  We can remove it when it is no longer relevant.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,16 +3,22 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "demo": "nx dev demo",
-    "build": "nx build demo",
-    "start": "nx start demo",
+    "dev": "nx dev next-demo",
+    "build": "nx build next-demo",
+    "start": "nx start next-demo",
+    "remix-demo": "nx dev demo",
```
Comment (stephanie56) [inline package.json:9]: Out of curiosity, are we keeping the remix build? Can we clean up the Remix related scripts?

**Review by stephanie56 — APPROVED** [review]
LGTM!

File: package.json (hunk 1 above)
Comment (brenzy) [reply package.json:9]: I wanted to keep it around for a little longer.  I think it makes sense to remove the Remix application if/when we start iterating on the demo application.

**Review by adamsullovey — APPROVED** [review]
At a glance, nothing Next.js-related needs changing.

**Pull request merged**